lib/rake/baseextensiontask.rb

```diff
@@ -35,7 +35,7 @@ def initialize(name = nil, gem_spec = nil)
     end
 
     def init(name = nil, gem_spec = nil)
-      @name = name
+      @name = name.to_s
```

Comment:
This change works only for `Task.new(name)` case.
We need to care about `Task#name=` case.

How about stringifying in `#define`?